gitlab/resource_gitlab_managed_license.go

```diff
@@ -0,0 +1,165 @@
+package gitlab
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/xanzy/go-gitlab"
+	"log"
+	"strconv"
+)
+
+func resourceGitlabManagedLicense() *schema.Resource {
+	return &schema.Resource{
+		Description: "This resource allows you to add rules for managing licenses on a project.\n" +
+			"For additional information, please see the " +
+			"[gitlab documentation](https://docs.gitlab.com/ee/user/compliance/license_compliance/).",
```

Comment:
Could you add a [markdown callout](https://www.terraform.io/registry/providers/docs#callouts) saying that this resource requires GitLab Ultimate?

Reply:
Reminder to generate the docs with `make reviewable` or simply `make generate` 😉

Reply:
Yeeeahp, I completely forgot and saw the prlint error in actions. Ah well, that's why CI is good~

Reply:
Also I previewed this change in this tool: https://registry.terraform.io/tools/doc-preview

And it needs two newlines `\n\n` between paragraphs.

In other resources I've done like

```go
Description: "My line 1\n\n" +
	"my line 2"
```